external/js/cothority/src/byzcoin/byzcoin-rpc.ts

```diff
@@ -107,17 +116,13 @@ export default class ByzCoinRPC implements ICounterUpdater {
 
         return rpc;
     }
-
+    latest: SkipBlock;
```

Comment:
private ?

Reply:
added a getter, as I want to read it out, so that I can store it and give it to a new `ByzCoinRPC` for faster startup.